open_academy/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+{
+    'name': "Open Academy",
```

Comment:
Please keep consistent if you will use single quote do it everywhere, same for double quotes, a good practice it's use double quotes.

Reply:
This change is pending. Please use double quotes

```suggestion
    "name": "Open Academy",
```